src/Experiment/tests/InterfaceAssemblyTest.cs

```diff
@@ -0,0 +1,217 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.CodeDom.Compiler;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Reflection;
+using System.Reflection.Emit.Experimental;
+using System.Reflection.Metadata;
+using System.Reflection.PortableExecutable;
+using System.Runtime.InteropServices;
+using Xunit;
+
+namespace Experiment.Tests
+{
+    public class InterfaceAssemblyTest
+    {
+        const bool _keepFiles = true; // keep files after testing for inspection
+        TempFileCollection _tfc;
+        string _assemblyFile;
+
+        [Fact]
+        public void Setup()
```

Comment:
This isn't the correct way to share assets between tests. See https://xunit.net/docs/shared-context.